apps/wallet-dashboard/components/dialogs/staking/views/DetailsView.tsx

```diff
@@ -154,6 +147,19 @@ export function DetailsView({
                             />
                         </div>
                     </Panel>
+                    {isAtRisk ? (
```

Comment:
We should also show this info for the active validators that are not in the committee 
`isAtRisk || !isValidatorCommitteeMember`